### PR TITLE
jupyterlab 3.4.4

### PR DESCRIPTION
**Pull request by Porkepix**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by Porkepix** [comment]
1H For an arm job, this will need a long build.

File: Formula/jupyterlab.rb [hunk 1]
```diff
@@ -158,8 +158,8 @@ class Jupyterlab < Formula
   end
 
   resource "jsonschema" do
```
Comment (SMillerDev) [inline Formula/jupyterlab.rb:160]: Can this be replaced by the formula?

Comment (Porkepix) [reply Formula/jupyterlab.rb:160]: What do you mean? You want it to `depends_on` on the formula? In which case do you use one or the other one?

Comment (SMillerDev) [reply Formula/jupyterlab.rb:160]: I want to try to depend on the jsonschema formula, yes.

Comment (Porkepix) [reply Formula/jupyterlab.rb]: Pushed the change; a new long build will be required though.

Comment (cho-m) [reply Formula/jupyterlab.rb]: This won't work without some `.pth` / PYTHONPATH manipulation since `jsonschema` is installed inside a virtualenv.

Comment (Porkepix) [reply Formula/jupyterlab.rb]: Then I've no idea about how this should be done.

**Pull request closed without merging**